src/components/subscriptions/TabContentTable.jsx

```diff
@@ -101,6 +104,27 @@ function TabContentTable({ enterpriseSlug }) {
     [users],
   );
 
+  const SubscriptionZeroStateMessaging = () => (
+    <div className="py-4 border shadow">
```

Comment:
Could we use the `Card` component here in place of explicitly using these padding, border, and shadow classes?

```jsx
<Card className="text-center">
  <Card.Body>
    <h2>Get Started</h2>
    <p className="py-2 lead">Assign your learners...</p>
    <AddUsersButton />
  </Card.Body>
</Card>
```